project/invites/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.core.fields import RichTextField
+
+from django.shortcuts import render
+import invitations.utils
+
+# Create your models here.
+class FriendInvitation(models.Model):
+    email = models.EmailField(max_length=254,blank=True,default=None)
+
+
+class InviteFriendsPage(Page):
+    max_count = 1
+
+    intro = RichTextField(blank=True,null=True,default="")
+    thankyou_page_title = models.CharField(
+        max_length=255, help_text="Title text to use for the 'thank you' page",blank=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('intro', classname="full"),
+        FieldPanel('thankyou_page_title'),
+    ]
+
+    def serve(self, request):
+
+        if request.method == 'POST':
+            Invitations = invitations.utils.get_invitation_model()
+            try:
+                invite_obj = Invitations.objects.get(email = request.POST.get("email"))
+                print(invite_obj.email)
+            except Invitations.DoesNotExist:
+                form = invitations.utils.get_invite_form()()
```

Comment:
The double parenthesis is subtle and may lead to unexpected behavior. Let's add a comment to emphasize we are doing two actions with one line here.

```suggestion
                # Get invitation form class and create instance
                form = invitations.utils.get_invite_form()()
```